contracts/base/LendingPool.sol

```diff
@@ -105,15 +106,15 @@ contract LendingPool is Base, LendingPoolInterface {
         tToken.burn(msg.sender, amount);
 
         // Withdraw the tokens from compound
-        _withdrawFromCompoundIfSupported(amount);
+        uint256 withdrawnAmount = _withdrawFromCompoundIfSupported(amount);
 
         // Transfers tokens
         tokenTransfer(msg.sender, amount);
 
         _markets().decreaseSupply(
             address(lendingToken),
             LoansInterface(loans).collateralToken(),
-            amount
```

Comment:
Should we include amount value in the event also? (for tracking purposes) So we know what amount was requested, and what actually was withdrawn.

Reply:
It makes sense to me. Just to keep the scope of this task, I created a new task to implement your change.